ojmicroline_thermostat/models/thermostat.py

```diff
@@ -36,43 +36,51 @@ class Thermostat:
     name: str
     online: bool
     heating: bool
-    adaptive_mode: bool
-    vacation_mode: bool
-    open_window_detection: bool
-    last_primary_mode_is_auto: bool
-    daylight_saving_active: bool
     regulation_mode: int
-    sensor_mode: int
-    temperature_floor: int
-    temperature_room: int
+    supported_regulation_modes: list[int]
     min_temperature: int
     max_temperature: int
-    temperatures: dict[int, int]
```

Comment:
I removed the `temperatures` dict in favor of individual fields (`comfort_temperature`, `boost_temperature`, etc.).

This felt better as it's closer to the data on the wire and allows the type system to express which temperatures *must* be present (`manual_temperature`, `comfort_temperature`) and which are optional (`vacation_temperature` is optional since WG4-thermostats don't support it).